### PR TITLE
LPS-128381 Adapt styles when a navigation link is a button

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by veroglez** [comment]
ci:test:sf

**Comment by veroglez** [comment]
ci:test:relevant

File: modules/apps/product-navigation/product-navigation-product-menu-theme-contributor/src/main/resources/META-INF/resources/product_navigation_product_menu.scss [hunk 1]
```diff
@@ -91,7 +91,8 @@
 }
 
 .product-menu {
-	a {
+	a,
+	button {
```
Comment (victorg1991) [inline modules/apps/product-navigation/product-navigation-product-menu-theme-contributor/src/main/resources/META-INF/resources/product_navigation_product_menu.scss:95]: I think we need to use a more specific class as this is applying a double border to some buttons:

Before:

Screenshot 2021-03-01 at 17 40 21

After:

Screenshot 2021-03-01 at 17 41 10

**Review by victorg1991 — CHANGES_REQUESTED** [review]

File: modules/apps/product-navigation/product-navigation-product-menu-theme-contributor/src/main/resources/META-INF/resources/product_navigation_product_menu.scss (hunk 1 above)
Comment (veroglez) [reply modules/apps/product-navigation/product-navigation-product-menu-theme-contributor/src/main/resources/META-INF/resources/product_navigation_product_menu.scss]: I have added a more specific class to solve this.

**Comment by victorg1991** [comment]
LGTM! :D

**Comment by victorg1991** [comment]
ci:forward

**Review by victorg1991 — APPROVED** [review]

**Pull request closed without merging**